### ISSUE TITLE
Add support for aliases

### ISSUE THREAD

**Issue by cbracken** [cbracken/dvm#3]
`dvm` currently supports a `default` environment. Add support for aliases. e.g.:

```
dvm alias create projectA 1.14.1
dvm alias create projectB 1.15.0
dvm use projectA --default
```

Like `gvm`, support the following sub-commands:

```
dvm alias create alias version
dvm alias delete alias
dvm alias list
```

where `version` can be another alias. 

This would mean adding an explicit environment file for each installed version. In order to avoid breakage when an alias is deleted, the environment files `dvm` creates should always specify paths in terms of the real underlying version, not any other alias.

**Comment by cbracken** [comment]
This will mean a per-environment `current` symlink and an overall `current` symlink that points to the current environment. Maybe something like:
```
~/.dvm/environments/current -> default
~/.dvm/environments/default/dart-sdk -> ~/../../darts/1.14.1
~/.dvm/darts/current -> ../environments/current/dart-sdk
```

Instead of the environment being a script, it'd be a directory under which an `env` file equivalent to today's environment script would exist.

**Comment by cbracken** [comment]
Something like `alias` would work too, and create an easy solution for #1 as well.

**Comment by cbracken** [comment]
Note, this likely allows a fairly simple solution to #1.

**Comment by cbracken** [comment]
Closed by 34e69b69bc096d90f386577b0ae19be6682bab8a.
